grails-app/migrations/views/stock-movement-list-item.sql

```diff
@@ -51,29 +51,29 @@ CREATE OR REPLACE VIEW stock_movement_list_item AS
         s.id AS shipment_id,
         s.current_status AS shipment_status,
         CASE
```

Comment:
I wonder if we should not move away from this part and move it to the StockMovement/OutboundStockMovement/StockMovementListItem classes, but I think this is part of @drodzewicz spike

Reply:
For now I just went with what was requested in the ticket, so to map the order statuses to different statuses than to what they were mapped until now.